<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>希両</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{
background-color: rgb(0, 0, 0);
border-radius:15px;
}
QScrollBar {width:0px;}
QScrollBar {height:0px;}
*:hover{
color: rgb(0, 170, 255);
}
#widget_show{
background-color: rgba(0,0,0,128); 
}
QMenu
{
background-color: rgba(0,0,0,128);
}
*{
font: 100 12pt &quot;华文仿宋&quot;;
border:0px;
min-height:30px;
color: rgb(255, 255, 255);
background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPixmapShow" name="pixmapshow" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_show" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_title" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>30</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>50</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>858</width>
                    <height>27</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_min">
                  <property name="text">
                   <string>最小化</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_close">
                  <property name="text">
                   <string>退出</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_lrcAcover" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_cover" native="true">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <property name="leftMargin">
                    <number>40</number>
                   </property>
                   <item>
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_back">
                     <property name="whatsThis">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;歌曲封面&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">*{
min-width:300px;
min-height:300px;
}</string>
                     </property>
                     <property name="text">
                      <string>封面</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_baseName">
                     <property name="styleSheet">
                      <string notr="true">*{
min-width:300px;
font: 14pt &quot;华文楷体&quot;;
}
</string>
                     </property>
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QScrollArea" name="scrollArea_lrc">
                  <property name="maximumSize">
                   <size>
                    <width>666</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents_lrc">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>666</width>
                     <height>495</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>666</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_lrc">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_others" native="true">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>20</number>
                </property>
                <property name="leftMargin">
                 <number>40</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>40</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_lrc_show">
                  <property name="text">
                   <string>词</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_loop">
                  <property name="whatsThis">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;切换循环模式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">*{
min-width:100px;
}
QComboBox
{
border:1px red solid;
}
QComboBox::drop-down 
{
width:0px;border:1px red solid;
}</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>顺序循环</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>随机循环</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>单曲循环</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>反序循环</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_value">
                  <property name="text">
                   <string>音量</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider_volume">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_table">
                  <property name="text">
                   <string>歌曲列表</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_seit_show">
                  <property name="text">
                   <string>设置</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_play_number" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>20</number>
                </property>
                <property name="leftMargin">
                 <number>40</number>
                </property>
                <property name="rightMargin">
                 <number>40</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_now">
                  <property name="text">
                   <string>当前</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider_play_value">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_end">
                  <property name="text">
                   <string>结束</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_play" native="true">
               <property name="styleSheet">
                <string notr="true">*{min-height:50px}</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_up">
                  <property name="text">
                   <string>上一首</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_stop">
                  <property name="text">
                   <string>暂停</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_play">
                  <property name="text">
                   <string>播放</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_down">
                  <property name="text">
                   <string>下一首</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPixmapShow</class>
   <extends>QWidget</extends>
   <header location="global">qpixmapshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
